tools/generators/ethereum/command.go.tmpl

```diff
@@ -148,16 +157,22 @@ func {{$contract.ShortVar}}{{$method.CapsName}}(c *cobra.Command, args []string)
         return err
     }
 
-    {{ range $i, $param := .ParamInfos }}
-    {{$param.Name}}, err := {{$param.ParsingFn}}(args[{{$i}}])
-    if err != nil {
+       {{ range $i, $param := .CmdArgInfos }}
```

Comment:
Looks like the indentation is still broken here.

Reply:
5a0562a46fea49b87a2abd144e2f74e3460967d0

Reply:
5a0562a46fea49b87a2abd144e2f74e3460967d0